tools/src/test/python/dlpx/virtualization/_internal/test_package_util.py

```diff
@@ -10,7 +10,7 @@
 class TestPackageUtil:
     @staticmethod
     def test_get_version():
-        assert package_util.get_version() == '3.2.0.dev0'
+        assert package_util.get_version() == '4.0.0.dev0'
 
```

Comment:
There is no need to do anything about this comment with this diff... but I wanted to raise a question for everyone...

Does this test have any utility whatsoever?

The existence of this test means the the developer workflow is often
1) Bump the version
2) Run unit tests
3) "Oh crap, I forgot, I also have to go and manually change that one test, too"
4) Change this test
5) Run unit tests again

I could be missing something, but steps (3-5) feel like unnecessary "makework".  It doesn't seem like this test is actually protecting us against anything, and so we could delete it, and make the release bump process smoother.

Reply:
I think it helps make sure someone did not update the version accidentally?

Reply:
I guess, but it seems like code reviews should handle that...

Or maybe the test could be modified to only check the "number" part of the version? Like, it probably matters that we don't accidentally move to 4.1.0.  But, if the dev part changes, that's probably not anything we need to test for, right?

Not a huge deal to me. Just thought I'd bring it up for discussion.